apps/portal/lib/graphql/resolvers/OldEvent.resolver.ts

```diff
@@ -25,25 +26,33 @@ export default class OldEventResolver {
   ): Promise<EventCheckin[]> {
     const event_data = await this.firebaseService.returnEventsbyProfile(netId, email);
 
-    const events = await this.eventService.findOldEventID(event_data);
+    if (event_data.length != 0) {
+      const events = await this.eventService.findOldEventID(event_data);
 
-    const profile = await context.prisma.profile.findFirst({
-      where: {
-        netid: netId,
-      },
-    });
+      const profile = await context.prisma.profile.findFirst({
+        where: {
+          netid: netId,
+        },
+      });
 
-    const reservations = await Promise.all(
-      events.map((event) =>
-        this.checkService.checkInEvent(
-          {
-            eventId: event.id,
-            profileId: profile.id,
-          },
-          context,
-        ),
-      ),
-    );
-    return reservations;
+      if (profile === null || profile === undefined) {
```

Comment:
Same as comment above. Also, I don't think we need an `else` statement there because of the `return` statement inside the if clause.